lib/util/trip/pending_ride_card.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_app/util/custom_timeline_theme.dart';
+import 'package:flutter_app/util/profiles/profile_widget.dart';
+import 'package:flutter_app/util/supabase.dart';
+import 'package:flutter_app/util/trip/trip_card.dart';
+import 'package:timelines/timelines.dart';
+import '../../drives/models/drive.dart';
+import '../../rides/models/ride.dart';
+import '../locale_manager.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class PendingRideCard extends TripCard<Ride> {
+  final Function() reloadPage;
+  final Drive drive;
+  const PendingRideCard(super.trip, {super.key, required this.reloadPage, required this.drive});
+
+  @override
+  Widget build(BuildContext context) {
+    Widget timeLine = FixedTimeline(
+      theme: CustomTimelineTheme.of(context),
+      children: [
+        TimelineTile(
+          contents: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Text('${localeManager.formatTime(trip.startTime)}  ${trip.start}'),
+              ],
+            ),
+          ),
+          node: const TimelineNode(
+            indicator: CustomOutlinedDotIndicator(),
+            endConnector: CustomSolidLineConnector(),
+          ),
+        ),
+        TimelineTile(
+          contents: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Text('${localeManager.formatTime(trip.endTime)}  ${trip.end}'),
+              ],
+            ),
+          ),
+          node: const TimelineNode(
+            indicator: CustomOutlinedDotIndicator(),
+            startConnector: CustomSolidLineConnector(),
+          ),
+        ),
+      ],
+    );
+
+    buildSeatsIndicator() {
+      Widget icon = Icon(
+        Icons.chair,
+        color: Theme.of(context).colorScheme.primary,
+      );
+      return Row(
+        mainAxisSize: MainAxisSize.min,
+        children: trip.seats <= 2
```

Comment:
this should be extracted into a widget (takes icon, text, and maybe cutoff limit) together with how we did it for the arrows

Reply:
i created a new icon not sure about the name though.

Reply:
fine by me, but maybe we will come up with a better name in the future